unittests/test_data/clair.json

```diff
@@ -68,10 +68,10 @@
               {
                 "Severity": "Unknown",
                 "NamespaceName": "pyupio",
-                "Link": "",
+                "Link": "https://access.redhat.com/errata/RHSA-2022:5311",
                 "FixedBy": "1.34.1-r3",
                 "Description": "TEST Patched vulnerability",
-                "Name": "pyup.io-42219 (PVE-2021-42219)",
+                "Name": "RHSA-pyup.io-42219 (PVE-2021-42219)",
```

Comment:
Maybe we shouldn't have the `RHSA` prefix in the name as that might be misleading for these vulns?

Reply:
Good point, updated.